core/ast_node.js

```diff
@@ -275,25 +276,29 @@ Blockly.ASTNode.prototype.findPreviousEditableField_ = function(location,
     var field = fieldRow[i];
     if (field.isCurrentlyEditable()) {
       previousField = field;
+      newAstNode = Blockly.ASTNode.createFieldNode(previousField);
```

Comment:
Can do:
`return Blockly.ASTNode.createFieldNode(previousField);`
here,
and
`return null`
below,

and leave out the declaration of `newAstNode` entirely.

Reply:
fixed